### PR TITLE
Fix linting issues

### PR DESCRIPTION
**Pull request by zygopleural**
I noticed that there were some linting issues: 
```shell
33 problems (33 errors, 0 warnings)
26 errors and 0 warnings potentially fixable with the `--fix` option.
```

So I have enabled checking and fixing those in the `package.json`.

I have ignored the `example` dir as I was seeing
```shell
The file does not match your project config: example/enums/Status.enum.ts.
The file must be included in at least one of the projects provided
```

I would suggest adding a pre-commit hook and / or a GHA CI workflow to check linting on PRs.

### CODE REVIEW

**Review by ridafkih — CHANGES_REQUESTED** [review]
Conflicts! 🕺

**Comment by zygopleural** [comment]
> Conflicts! 🕺

On it

**Comment by zygopleural** [comment]
> > Conflicts! 🕺
> 
> On it

Done

File: .eslintrc [hunk 1]
```diff
@@ -5,7 +5,6 @@
     "plugin:prettier/recommended"
   ],
   "plugins": ["@typescript-eslint"],
-  "files": ["*.ts"],
```
Comment (zygopleural) [inline .eslintrc:8]: This is actually not a valid config option. Dunno why I originally added it.

**Review by ridafkih — APPROVED** [review]

**Pull request merged**